graylog2-server/src/main/java/org/graylog2/rest/resources/system/MetricsHistoryResource.java

```diff
@@ -58,129 +42,19 @@
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
-import java.util.SortedSet;
 
 @RequiresAuthentication
-@Api(value = "System/Metrics", description = "Internal Graylog2 metrics")
-@Path("/system/metrics")
-public class MetricsResource extends RestResource {
-
-    private static final Logger LOG = LoggerFactory.getLogger(MetricsResource.class);
-    private final MetricRegistry metricRegistry;
+@Api(value = "System/Metrics/History", description = "Get history of metrics")
+@Path("/system/metrics/{metricName}/history")
+@Produces(MediaType.APPLICATION_JSON)
+public class MetricsHistoryResource extends RestResource {
     private final MongoConnection mongoConnection;
 
     @Inject
-    public MetricsResource(MetricRegistry metricRegistry,
-                           MongoConnection mongoConnection) {
-        this.metricRegistry = metricRegistry;
+    public MetricsHistoryResource(MongoConnection mongoConnection) {
```

Comment:
Why did you rename the resource and remove all those other methods? They're being used by graylog2-web-interface.